code/game/gamemodes/clown_ops/clown_ops.dm

```diff
@@ -58,6 +58,7 @@
 	if(visualsOnly)
 		return
 	H.dna.add_mutation(SMILE)
+	H.faction |= "clown"
```

Comment:
shouldn't this be LAZYADD and not a bitwise operation since its adding a string to a list or does this actually work?